CME/admin/components/Credit/Edit.php

```diff
@@ -125,8 +123,6 @@ protected function initFrontMatter()
 					)
 				);
 			}
-		} else {
-			$this->front_matter = $this->credit->front_matter;
```

Comment:
Since you're removing the `$this->front_matter` property, it should also be removed from the protected properties at the top of the file.